docs/contributing/documentation_guidelines.md

```diff
@@ -190,7 +222,7 @@ Only use tables when needed, with the “simple” reStructuredText syntax, whic
 
 <summary>Rendered output</summary>
 
-```eval_rst
+```{eval-rst}
 =============  =============
 Browser        Device/OS    
 =============  =============
```

Comment:
Reported here https://github.com/executablebooks/MyST-Parser/issues/534.

Additionally I think it would be nice for us to move to GFM Markdown syntax for tables, rather than eval-rst.